_topic_map.yml

```diff
@@ -2022,6 +2022,9 @@ Topics:
 - Name: Deploying distributed units at scale in a disconnected environment
   File: ztp-deploying-disconnected
   Distros: openshift-webscale
+- Name: NBDE implementation guide
+  File: ztp-nbde-implementation-guide
```

Comment:
This implementation guide isn't ZTP-specific, but generic to all OpenShift implementations.

Reply:
Agreed. If you're referring to the filename, that just makes it easy for me to track in my repo. It doesn't appear in the document.